gdal/doc/gdal_tutorial.dox

```diff
@@ -57,11 +57,10 @@ int main()
 
 In Python:
 \code
-    import gdal
-    from gdalconst import *
+    from osgeo import gdal, gdalconst
 
-    dataset = gdal.Open( filename, GA_ReadOnly )
-    if dataset is None:
+    dataset = gdal.Open(filename, gdalconst.GA_ReadOnly)
```

Comment:
personnaly, I never use gdalconst but just gdal. So gdal.GA_ReadOnly. That's probably easier for users to use only the gdal module